controllers/team.controller.js

```diff
@@ -71,8 +71,24 @@ function createdTeam(req, res) {
     });
 }
 
+/**
+ * @function createdTeam
+ * @param {*} req
```

Comment:
Copy paste mistakes? ;)

Reply:
Nope, req doesn't need to have anything!

Reply:
Oh rip, you got me, `@function createdTeam`